infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/AWSProvider.java

```diff
@@ -100,9 +102,56 @@ public boolean canHandle(InfrastructureConfig infrastructureConfig) {
     }
 
     @Override
-    public void init() throws TestGridInfrastructureException {
+    public void init(TestPlan testPlan) throws TestGridInfrastructureException {
         cfScriptPreprocessor = new CloudFormationScriptPreprocessor();
         amiMapper = new AMIMapper();
+        String scriptsLocation = testPlan.getScenarioTestsRepository();
+        ShellExecutor shellExecutor = new ShellExecutor(Paths.get(scriptsLocation));
+
+        if (testPlan.getScenarioConfig().getScripts() != null
+                && testPlan.getScenarioConfig().getScripts().size() > 0) {
+            for (Script script : testPlan.getScenarioConfig().getScripts()) {
+                if (Script.Phase.CREATE.equals(script.getPhase())) {
+                    try {
+                        logger.info("Provisioning additional infra");
```

Comment:
Which file are we running? which phase? It's better to log all these.
See the fourth commandment in here -
http://www.masterzen.fr/2013/01/13/the-10-commandments-of-logging/ :)